### ISSUE TITLE
dynavers_modules path relative to cwd, not the config file

### ISSUE THREAD

**Issue by jan-molak** [numical/dynavers#1]
Hey @numical and many thanks for your work on dynavers!

I'd like to use it to test one of my projects, but I'm facing a difficulty caused by the fact that the `path` to `dynavers_modules` specified in the `dynavers.json` is relative to the process `cwd`, rather than the config file itself. Could this behaviour be changed/made configurable by any chance?

In my scenario, the (simplified) directory layout looks more or less like this:

```
|- spec
|  |- integration
|  | |- module-under-test-tests
|  |    |- test.js
|  |- dynavers.json
|- package.json
```

With a following `dynavers.json`, the `dynavers_modules` directory will be created at the same level as `package.json` (because the installation is invoked via npm):
```
{
  "path": "dynavers_modules",
  "versions": {
    "module-under-test": ["1.0.0", "2.0.0"]
  }
}
```

This wouldn't be a problem if the `test.js` was also invoked from the top level directory, but for various reasons it can't be. The cwd of the `test.js` is `spec/integration/module-under-test-tests`

Now, should the `test.js` try to invoke:
```
const setModuleVersion = require('dynavers')('../dynavers.json')
setModuleVersion('module-under-test', '1.0.0')
```

Dynavers would complain with the following error message:

```
dynavers_modules/module-under-test-1.0.0: No such file or directory. Run `load-module-versions` to install.
```

To my current understanding, this happens because `setModuleVersion` tries to look for `dynavers_modules` under `spec/integration/module-under-test-tests` not at the top level.

I think if the `path` pointed to a directory relative to the configuration file rather than the `process.cwd()`the problem would be solved.

I'm looking forward to hearing your thoughts.

Best,
Jan

**Comment by numical** [comment]
Hi Jan,
Sorry for my delayed response.
Your suggestion seems reasonable enough - there's no reason not to make the module paths more flexible.
How about changing [line 20 of index.js](https://github.com/numical/dynavers/blob/master/index.js#L20) to something like:
```javascript
var modulePath = path.join(__dirName, spec.path, moduleName + '-' + version);
```
That way you simply specify the plugin-relative path in the spec.
I'm not doing it immediately as I must think about backward-compatibility - and also I'm snowed with other work - but try this out for your use case and see if it is sufficient.
If good, I'll see when I get it into an 'official' release.

**Comment by numical** [comment]
Hi @jan-molak - did you try my suggestion above?  Did it help?

**Comment by jan-molak** [comment]
Hi @numical, apologies for the delay in getting back to you. 

I decided to take a different approach. I started using [Lerna](https://github.com/lerna/lerna), which allows me to create different "adapter" modules corresponding to different versions of my project's dependencies. As each adapter is a stand-alone npm module, each has a `package.json` of its own. This helps to use different major versions of the same dependency within the same multi-module project.

Many thanks for your help and suggestions!

J
